### PR TITLE
DOCSP-38222: primer

### PR DESCRIPTION
**Pull request by rustagir**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-java-rs/blob/master/REVIEWING.md)

**Note: this is part of a consolidation effort, so your review should
not address style guide issues or phrasing unless there are errors**

JIRA - <https://jira.mongodb.org/browse/DOCSP-38222>
Staging - <https://preview-mongodbrustagir.gatsbyjs.io/scala/DOCSP-38222-primer/get-started/primer/>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/get-started/primer.txt [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+This guide provides background about the {+driver-short+} and its
+asynchronous API before showing you how to use the driver and MongoDB in
+the 
+
+.. :ref:`Quick Start guide <>`.
```
Comment (jordan-smith721) [inline source/get-started/primer.txt:24]: Just verifying that this ref is commented out (leaving an unfinished sentence) on purpose?

File: source/get-started/primer.txt [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+   To learn more about reactive streams, visit
+   the `Reactive Streams documentation <http://www.reactive-streams.org>`__.
+
+Observables
+-----------
+
+The {+driver-short+} API mirrors the :driver:`Java Sync driver
+</java/sync/current/>` API and any methods that cause network 
+I/O to return a ``Observable<T>`` type, where ``T`` is the type of response
```
Comment (jordan-smith721) [inline source/get-started/primer.txt:73]:
```suggestion
I/O to return an ``Observable<T>`` type, where ``T`` is the type of response
```

File: source/get-started/primer.txt [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+for the operation.
+
+.. note::
+
+   All ``Observable`` types returned from the API are
+   `cold <https://projectreactor.io/docs/core/release/reference/#reactor.hotCold>`__,
```
Comment (jordan-smith721) [inline source/get-started/primer.txt:79]: This link is broken (possibly temporarily based on the error message)

**Review by jordan-smith721 — APPROVED** [review]
LGTM w/ a couple things

File: source/get-started/primer.txt (hunk 3 above)
Comment (rustagir) [reply source/get-started/primer.txt:79]: I think it is a site maintenance period. I just tested the link yesterday but I'll make sure to test in the next few days to see if the site is up again!

**Pull request merged**